### DISCUSSION TITLE
v1.7.2rc2

### DISCUSSION THREAD

**Discussion by schlich** [dbt-msft/dbt-sqlserver#465] [General]
## What's Changed
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/dbt-msft/dbt-sqlserver/pull/404
* Incorporate 1.7.2 changes by @cody-scott in https://github.com/dbt-msft/dbt-sqlserver/pull/461

## New Contributors
* @cody-scott made their first contribution in https://github.com/dbt-msft/dbt-sqlserver/pull/461

**Full Changelog**: https://github.com/dbt-msft/dbt-sqlserver/compare/v1.4.3...v1.7.2rc1

<hr /><em>This discussion was created from the release <a href='https://github.com/dbt-msft/dbt-sqlserver/releases/tag/v1.7.2rc1'>v1.7.2rc1</a>.</em>

**Comment by ericmuijsvanoord** [comment]
Intial testing: Its working very well on our dbt project here. I will do some further testing this week. 

Thanks so much @cody-scott  @schlich for the ton of work! 
What is required to get it to a full release? More testing?

**Reply by schlich** [reply]
Release will go out in the next couple of days 🚀
